src/main/java/gregtech/api/interfaces/metatileentity/IMetaTileEntityCable.java

```diff
@@ -9,5 +9,7 @@
     @Deprecated
     public long transferElectricity(byte aSide, long aVoltage, long aAmperage, ArrayList<TileEntity> aAlreadyPassedTileEntityList);
 
-    public long transferElectricity(byte aSide, long aVoltage, long aAmperage, HashSet<TileEntity> aAlreadyPassedSet);
+    default public long transferElectricity(byte aSide, long aVoltage, long aAmperage, HashSet<TileEntity> aAlreadyPassedSet) {
```

Comment:
Default should not be used as it prevents the mod from being compiled for 1.7, meaning it will not be useable on some systems.  Instead a secondary class with static functions should contain these functions instead, to be called directly in the implementations of this interface.  This is, consequently, exactly how Java 8 compiles it internally anyway.